engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-action-packages.controller.js

```diff
@@ -84,5 +84,22 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkActionPackag
             return deferred.promise;
         };
 
+        $scope.performViaRemoteExecution = function(action, customize) {
+            var remoteAction, selectedHosts = $scope.nutupane.getAllSelectedResults();
+            if ($scope.content.contentType == "package_group") {
+                remoteAction = "group_" + action;
+            } else if ($scope.content.contentType == "package") {
+                remoteAction = "package_" + action;
+            }
+            form = $('#packageActionForm');
+            form.attr('action', '/katello/remote_execution');
+            form.attr('method', 'post');
```

Comment:
Because I want to get to the remote execution UI after using the UI feature. Using the API and then using `$window.location.href` to getting to the remote execution screen, is just a more complicated way of doing so as we do right now (+ we loose the ability to show the form with invalid data, where there are some)